### PR TITLE
Fix failed e2e test, update npm packages

### PR DESCRIPTION
**Pull request by kienstra**
## Changes
* Fix failed e2e test 
* Update npm packages
* Remove some commands that aren't needed anymore
* Remove needless patch versions in orbs

## Testing instructions
Not needed, just a quick sanity check would be great

### CODE REVIEW

File: js/src/edit-block/hooks/useEditor.js [hunk 1]
```diff
@@ -29,7 +29,6 @@ const useEditor = () => {
 	 * @param {Object} post Post object.
 	 */
 	const setupEditor = ( post ) => {
-		resetPost( post );
```
Comment (kienstra) [inline js/src/edit-block/hooks/useEditor.js:32]: Calling this caused the warning that failed the e2e tests

File: .circleci/config.yml [hunk 2]
```diff
@@ -51,7 +50,6 @@ jobs:
           MYSQL_ROOT_PASSWORD: wordpress
     steps:
       - checkout
-      - run: sudo apt-get update && sudo apt-get install php-mbstring
```
Comment (kienstra) [inline .circleci/config.yml:54]: Luckily not needed anymore

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 version: 2.1
 
 orbs:
-  node: circleci/node@5.0.0
-  php: circleci/php@1.1.0
```
Comment (kienstra) [inline .circleci/config.yml:5]: If you leave out the patch version, it gets the latest patch version

**Review by mike-day — APPROVED** [review]
Nice work cleaning up! Dev environment seems to be working as expected for me, and formerly failing e2e tests are passing locally:

Screen Shot 2022-08-03 at 5 55 01 PM

**Comment by kienstra** [comment]
Thanks a lot, @mike-day!

**Pull request merged**